src/components/02-components/card-number/card-number.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+
+export type BrandProps = {
```

Comment:
Why `BrandProps`? Use the right name for this type, based on the component name.